lib/client-bundle-plugin.js

```diff
@@ -137,10 +137,24 @@ export default function (inputOptions, outputOptions, resolveFileUrl) {
       const dependencies = getDependencies(clientOutput, clientEntry);
 
       if (property.startsWith(allSrcPlaceholder)) {
-        const depCodes = dependencies.map(
-          (name) => clientOutput.find((item) => item.fileName === name).code,
-        );
-        return JSON.stringify([clientEntry.code, ...depCodes].join(';'));
+        const allModules = [
+          clientEntry,
+          ...dependencies.map((name) =>
+            clientOutput.find((item) => item.fileName === name),
+          ),
+        ];
+
+        const inlineDefines = [
+          ...allModules.map(
+            (item) =>
+              `nextDefineUri=location.origin+${resolveFileUrl(item)};${
```

Comment:
@jakearchibald Quick question: here I'm just calling `resolveFileUrl(item)` since our `resolveFileUrl` only needs `fileName` property anyway, but I see that other calls in the same file use

```js
        return resolveFileUrl({
          fileName: clientEntry.fileName,
          moduleId,
          format,
        });
```

or

```js
            return resolveFileUrl({
              fileName: entry.fileName,
              moduleId,
              format: outputOptions.format,
            });
```

instead of just passing object directly.

Should I do the same?

Reply:
I think it's fine. If we change format there'll be a lot of refactoring anyway.